src/Monolog/Handler.php

```diff
@@ -47,6 +47,7 @@ protected function write(array $record): void
         $payload = [
             'level' => $this->getSeverityFromLevel($record['level']),
             'message' => $record['message'],
+            'logger' => sprintf('monolog.%s', $record['channel']),
```

Comment:
I was thinking about adding just channel name, but decided to add it this way - `monolog.$channel` imho it is better, feedback is appreciated.

Reply:
This seems sensible to me :+1:

Reply:
I fail to see a good reason to force the prefix on a user-submitted value. If he wants to distinguish between an error logged from a "plain" client or from Monolog then he can just change the `logger` property (in this case by setting the Monolog channel)

Reply:
Since the property is named `logger` and not `channel` i though that knowing it was logged via monolog + combining with channel's name might be really helpful - at least for me it has added value.

If we take away this prefix, most people (it is the most common scenario that people uses monolog in some framework, etc Symfony, and channel name is `default`) will see only `default` vs `monolog.default`. I would be thinking about what the hell does the `default` mean, when i see `monolog.default` i know that it is monolog channel on first sight.

Reply:
Got it, just not really happy as I was saying to force something on user's behalf. By the way let's merge it as is and let's see how it goes